docs/usage/types.md

```diff
@@ -441,6 +441,9 @@ _(This script is complete, it should run "as is")_
 `PostgresDsn`
 : a postgres DSN style URL; see [URLs](#urls)
 
+`RabbitmqDsn`
+: a rabbitmq DSN style URL; see [URLs](#urls)
```

Comment:
You need something saying the scheme is `amqp`. Presumably there are other reasons beyond rabbitmq that someone in python might want to use AMQP. I'm happy not to change the name, but I think we should have a brief commenting "... or any other application where you need AMQP" or something.

Reply:
partly just to aid people searching for amqp not rabbitmq